framework/views/migration.php

```diff
@@ -18,7 +18,7 @@
 use yii\db\Migration;
 
 /**
```

Comment:
Let's remove the docblock altogether. No need for it.

Reply:
remove docblock lines